src/calendar/tests/functional/Calendar.ts

```diff
@@ -0,0 +1,172 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import * as keys from 'leadfoot/keys';
+import * as css from '../../styles/calendar.m.css';
+
+const DELAY = 500;
+
+function openMonthPicker(remote: any) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=calendar')
+		.setFindTimeout(5000)
+		.findByTagName('button')
+			.click()
+			.end();
+}
+
+function clickDate(remote: any) {
+	const today = new Date();
+	const firstDay = new Date(today.getFullYear(), today.getMonth(), 1).getDay();
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=calendar')
+		.setFindTimeout(5000)
+		.findByCssSelector(`tbody > tr:first-child > td:nth-child(${firstDay + 1})`)
+			.click()
```

Comment:
Should I indent the `.then()` blocks? I've mostly been going off formatting in other widget functional tests.